lang/cs/Org.Apache.REEF.Client/Common/HttpClientHelper.cs

```diff
@@ -90,12 +108,17 @@ internal static List<string> GetRmUri(string filePath)
             }
         }
 
-        internal static string GetAppId(string filePath)
+        /// <summary>
+        /// Get application id for the Job submitted 
+        /// </summary>
+        /// <param name="filePath"></param>
+        /// <returns></returns>
+        internal string GetAppId(string filePath)
         {
             using (var sr = new StreamReader(File.Open(filePath, FileMode.Open, FileAccess.Read, FileShare.Read)))
             {
-                var appId = sr.ReadLine();                
-                return appId;
+                _appId = sr.ReadLine();
```

Comment:
Those were original code. We should clean up to read the data only once and leave them in variables. They are fixed per job submission. We can either have a separate Jira to do it, or as it is Unstable, we might change it soon with rest API submission.